packages/wonder-blocks-tooltip/components/tooltip-anchor.test.js

```diff
@@ -0,0 +1,560 @@
+            "addEventListener",
+        );
+        addEventListenerSpy.mockClear();
+
+        // Act
+        mount(nodes);
+
+        // Assert
+        expect(addEventListenerSpy).toHaveBeenCalledWith(
+            "focusin",
+            expect.any(Function),
+        );
+        expect(addEventListenerSpy).toHaveBeenCalledWith(
+            "focusout",
+            expect.any(Function),
+        );
+        expect(addEventListenerSpy).toHaveBeenCalledWith(
+            "mouseenter",
+            expect.any(Function),
+        );
+        expect(addEventListenerSpy).toHaveBeenCalledWith(
+            "mouseleave",
+            expect.any(Function),
+        );
+    });
+
+    test("on unmount, unsubscribes from focus and hover events", () => {
+        // Arrange
+        const getFakeTooltipPortalMounter = (active) =>
+            (((
+                <div id="anchor">{active ? "true" : "false"}</div>
+            ): any): React.Element<typeof TooltipPortalMounter>);
+        const nodes = (
+            <div>
+                <TooltipAnchor anchorRef={() => {}}>
+                    {getFakeTooltipPortalMounter}
+                </TooltipAnchor>
+            </div>
+        );
+        const removeEventListenerSpy = jest.spyOn(
+            HTMLElement.prototype,
+            "removeEventListener",
+        );
+        removeEventListenerSpy.mockClear();
+        const wrapper = mount(nodes);
+
+        // Act
+        wrapper.unmount();
+
+        // Assert
+        expect(removeEventListenerSpy).toHaveBeenCalledWith(
+            "focusin",
+            expect.any(Function),
+        );
+        expect(removeEventListenerSpy).toHaveBeenCalledWith(
+            "focusout",
+            expect.any(Function),
+        );
+        expect(removeEventListenerSpy).toHaveBeenCalledWith(
+            "mouseenter",
+            expect.any(Function),
+        );
+        expect(removeEventListenerSpy).toHaveBeenCalledWith(
+            "mouseleave",
+            expect.any(Function),
+        );
+    });
+
+    describe("forceAnchorFocusivity is true", () => {
+        test("if not set, sets tabindex on anchor target", (done) => {
+            // Arrange
+            const arrange = (actAndAssert) => {
+                const fakeTooltipPortalMounter = (((
+                    <div>This is the anchor</div>
+                ): any): React.Element<typeof TooltipPortalMounter>);
+                const nodes = (
+                    <div>
+                        <TooltipAnchor
+                            forceAnchorFocusivity={true}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </div>
+                );
+                mount(nodes);
+            };
+
+            const actAndAssert = (ref) => {
+                if (!ref) {
+                    return;
```

Comment:
Does this ever happen?  If so won't the test time out.  Should we may be have
```
expect(true).toBe(false);  // or some other way to force a test failure
done();
```

Reply:
It's just to shut `flow` up. I'll change it to be less obvious.

Reply:
`more obvious`?